introduction_to_amazon_algorithms/forecasting_services_comparison/SageMaker_DeepAR_Amazon_Forecast.ipynb

```diff
@@ -0,0 +1,1938 @@
+{
```

Comment:
<p>Double check that Ipython.display works as expected on SageMaker Studio. Seem to remember there's some incompatibility, but that might be fixed by now.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3523/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>